### PR TITLE
Development

### PR DESCRIPTION
**Pull request by dakenblack**

### CODE REVIEW

**Comment by tommitytom** [comment]
There are quite a few linting errors in this PR - We use linting to make sure that our code styling is easy readable and consistent across all files. You can seen linting errors by typing "gulp vet".  I have also just pushed a commit which makes vetting compulsory - so now whenever you compile for distribution you will see the linting errors and the compile will fail.  Also if you use SublimeText it's worth checking out SublimeLint which shows you linting errors while you are coding :)

**Comment by dakenblack** [comment]
Sure I'll fix it and let you know

**Comment by dakenblack** [comment]
done

File: src/Project.js [hunk 1]
```diff
@@ -230,8 +230,11 @@ export default class Project extends SkyGridObject {
 	 * @param  {Boolean} [fetch]		Determines whether the full user object should be fetched, or just the description.  Defaults to true.
 	 * @returns {Promise<User[], SkyGridError>} A promise that resolves to an array of all users that were found.
 	 */
-	users(constraints, fetch = true) {
-		return this._api.request('findUsers', { 
+	users(constraints, fetch) {
```
Comment (tommitytom) [inline src/Project.js]: Here's an example of it in action, with a sample of the transpiled code:

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&code=class%20Foo%20%7B%0A%20%20%20bar(baz%20%3D%201)%20%7B%0A%20%20%20%20console.log(baz)%3B%0A%20%20%7D%0A%7D%0A%0Aconst%20foo%20%3D%20new%20Foo()%3B%0Afoo.bar()%3B

**Comment by dakenblack** [comment]
hmmm I seem to have imagined the issue, I've reverted the commit

**Pull request merged**